src/main/java/com/scottlogic/AuthorPostSorter.java

```diff
@@ -0,0 +1,21 @@
+package com.scottlogic;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class AuthorPostSorter implements PostSorter {
+
+    public List<UserPost> sort(List<UserPost> inputList) {
+        if (inputList == null) {
+            return Arrays.asList();
+        }
+
+        if (inputList.size() == 0) {
+            return inputList;
+        }
+        Collections.sort(inputList, Comparator.comparing(UserPost::getAuthor));
+        return inputList;
```

Comment:
Simple enough to do
`return Collections.sort(inputList, Comparator.comparing(UserPost::getAuthor))`

Reply:
If you do a static import for Comparator.comparing, you won't need the 'Comparator' part

Reply:
Fixed the return. 

As for the static import, I attempted it, however, it threw a 'Cannot resolve symbol' error after adding the 'static' part - looked at numerous resources alas nothing helped. Think I may keep this for now till I know IntelliJ a little better?

Reply:
I'm assuming you solved the static import part from our messages on Slack